### DISCUSSION TITLE
june 19, how are we gonna do it?

### DISCUSSION THREAD

**Discussion by idontveapc** [Docile-Alligator/Infinity-For-Reddit#1457] [Q&A]
api will require fee, and i don't know that what would happen on 19 june? update or no reddit? :(

**Comment by MinekPo1** [comment]
A possible solution is to allow users to use a custom app token, as reddit promised to keep bots working. An alternative and more demanding solution is to use the api reddit uses on their mobile app, but this solution likely requires deeper changes.
